.github/workflows/build-push-dockerhub.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: build-and-publish-dockerhub
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build-push:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: build image
+        shell: bash
+        run: |
+          docker build -t jackdwyer/bom_exporter:latest .
```

Comment:
I've configured a repository variable "DOCKERHUB_TAG" to parameterise this, please update.
It points to 'gkoh0/bom_exporter:latest'.
Whilst I would happily use your DockerHub repository:
* managing secrets would be messy
* people might get suspicious about the github/dockerhub username mismatch